pep-0622.rst

```diff
@@ -502,6 +580,12 @@ considered for constant value patterns.
 Sequence Patterns
 ~~~~~~~~~~~~~~~~~
 
+Simplified syntax::
+
+  sequence_pattern: '[' [values_pattern] ']' | '(' [values_pattern] ')'
+  values_pattern: ','.value_pattern+ ','?
+  value_pattern: '*' capture_pattern | pattern
```

Comment:
There's a lie here -- `(foo)` is not a sequence pattern, but `[foo]` and `(foo,)` are. Not sure how to fix without adding another helper rule (like in the Appendix).

Reply:
`'(' [value_pattern ',' [values pattern]] ')'`?

Reply:
Ah, should we add another alt for parenthesized tuples?

`'[' [values_pattern] ']' | '(' [value_pattern ',' [values pattern]] ')' | value_pattern ',' [values pattern]`

Reply:
I would leave that out here. In the real grammar it's handled differently. If we draw more attention to it here we'll just get complaints about ambiguity in `MyClass(p1, p2)`. Maybe a note in the text is better?

Reply:
I'll add a note on the first simplified grammar saying "this is for readability, go to the appendix for real specs". In nicer words.